### DISCUSSION TITLE
Welcome to Perfect Pearls Discussions!

### DISCUSSION THREAD

**Discussion by cobaltt7** [RedGuy12/perfect-pearls#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by return-misaPuding** [comment]
Is your Scratch account RedGuy7?

**Reply by cobaltt7** [reply]
Yes
